docs/advanced/meta_opt.rst

```diff
@@ -1,252 +0,0 @@
-*****************************************
```

Comment:
Why remove?

Reply:
I see, I think we should provide a link to Ray tune somewhere in our documentation so when users have a need, they can know the optimal solution.

Reply:
I don’t see why do we need to do it. Ray has lots of other components. Why do we need to provide a link to this particular one?